src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -139,6 +147,15 @@ int main(int argc, const char *argv[]) {
 
   EmscriptenGlueGenerator generator(wasm);
   generator.fixInvokeFunctionNames();
+
+  if (legalizeJavaScriptFFI) {
+    PassRunner passRunner(&wasm);
+    passRunner.setDebug(options.debug);
+    passRunner.setDebugInfo(debugInfo);
```

Comment:
Is debugInfo actually used in that pass? Make sense to have it though for future-proofing I suppose.